src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -226,6 +228,10 @@ const SignOrExecuteForm = ({
 
         {canExecute && <ExecuteCheckbox checked={shouldExecute} onChange={setShouldExecute} disabled={onlyExecute} />}
 
+        {shouldExecute && (
+          <ExecutionMethod walletLabel={wallet?.label || ''} willRelay={willRelay} onChange={setWillRelay} />
```

Comment:
Nit: I would not pass the state setter here but the actual `onChange` handler you have in the child.

Reply:
I've applied the suggestion. And I also changed the state variable type from boolean to `ExecutionType`.

I've run into a type issue... Is there a way around this casting?
https://github.com/safe-global/web-core/blob/bb1c5cc137ecbd154bc3ecd9529c0327806f9681/src/components/tx/ExecutionMethod/index.tsx#L107